tests/CMakeLists.txt

```diff
@@ -1400,6 +1402,11 @@ add_custom_target(issue2342 ALL
   WORKING_DIRECTORY ${GEN_SCRIPT_DIR}
   COMMENT "Generating issue2342.scad"
 )
+add_custom_target(svg_viewbox_tests ALL
+  COMMAND ${GEN_SCRIPT_DIR}/gen_svg_viewbox_tests.py "${CMAKE_SOURCE_DIR}/../testdata/svg/viewbox" "${CMAKE_SOURCE_DIR}/../testdata/scad/svg"
```

Comment:
I think running this with `${PYTHON_EXECUTABLE}` would be safer. For some reason, the Python executable in my path is actually a shell script (due to me using `pyenv` to manage python). Using shell scripts in shebangs seems to not work as expected..

Reply:
It is, the files is generated as the issue2342 script. Not sure if that's the best solution or just running it explicitely with the python executable is simpler.

Reply:
Yup, we can always deal with this separate from this PR, as the other python script started breaking for me as well : /